src/app/modules/create-facility/pages/home/home.component.html

```diff
@@ -1,36 +1,16 @@
 <common-page-framework layout='blank'>
-  <h1 class="border-bottom">{{pageTitle}}</h1>
-
-  <p>
-    This form is to be completed by applicants for facilities that do not have a valid
-    Medical Services Plan Facility Number.
-  </p>
-
-  <p>
-    All information on this form must be authorized by the responsible Practitioner for the facility
-    (the "Administrator" to which the Medical Services Plan Facility Number may be issued. If there are
-    any changes to the information contained in thiIn addition to applying for a Medical Services Plan Facility Number you will also be able to request that the Business Cost Premium be applied to the facility.tails for Facilities with an MSP Facility Numbe' form.
-  </p>
-
-
-  <p>
-    To complete the Application for Medical Services Plan Facility Number form you will require the following pieces of information:
-  </p>
-  <ul>
-    <li>
-      Facility administrator information including name and Medical Services Plan practitioner number
-    </li>
-    <li>
-      Facility information including facility name, address and postal code
-    </li>
-  </ul>
-  <p>
-    In addition to applying for a Medical Services Plan Facility Number you will also be able to request that the
-    Business Cost Premium be applied to the facility.
-  </p>
+<h1 class="border-bottom">{{pageTitle}}</h1>
+  <hr>
 
+    <p>This form is to be completed by applicants for facilities that do not have a valid Medical Services Plan Facility Number.</p>
+    <p>All information on this form must be authorized by the responsible Practitioner for the facility (the "Administrator") to which the Medical Services Plan Facility Number may be issued. If there are any changes to the information contained in this form after it has been submitted, the Administrator must submit an 'Application to Cancel or Change Details for Facilities with an Medical Services Plan Facility Number' form.</p>
+    <p>To complete the Application for Medical Services Plan Facility Number form you will require the following pieces of information:</p>
+    <ul>
+      <li>Facility administrator information including name and Medical Services Plan practitioner number</li>
+      <li>Facility information including facility name, address and postal code</li>
+    </ul>
+    <p>In addition to applying for a Medical Services Plan Facility Number you will also be able to request that the Business Cost Premium be applied to the facility.</p>
 </common-page-framework>
-
 <common-form-action-bar [defaultColor]='true'
```

Comment:
Remove common-form-action-bar - it now resides in the container for this module

Reply:
Removed.